Mathlib/Algebra/Order/Group/Int.lean

```diff
@@ -34,8 +34,10 @@ open Function Nat
 
 namespace Int
 
-theorem coe_nat_strictMono : StrictMono (· : ℕ → ℤ) := fun _ _ ↦ Int.ofNat_lt.2
-#align int.coe_nat_strict_mono Int.coe_nat_strictMono
+theorem natCast_strictMono : StrictMono (· : ℕ → ℤ) := fun _ _ ↦ Int.ofNat_lt.2
```

Comment:
Does the naming convention specify the order of words here, i.e., `natCast_strictMono` or `strictMono_natCast` (where the latter follows more closely the order of symbols in the statement)? Greping mathlib, I found several examples in both directions.

Reply:
`injective`, `mono`, `strictMono` go to the end. Most other predicates go to the front.

Reply:
I could not find comments about this in https://leanprover-community.github.io/contribute/naming.html. Could you maybe update this page (and try to explain the convention, which looks pretty arbitrary to me)?

Reply:
I will do that, but can we not hold up this PR on it? It's completely orthogonal.